microsoft/testsuites/vm_extensions/vmsnapshot_extension.py

```diff
@@ -100,6 +108,51 @@ def verify_vmsnapshot_extension(
                     pass
                 else:
                     raise e
-            time.sleep(1)
+            time.sleep(5)
```

Comment:
keep 1 second is better. If it needs to wait some condition, use a loop to check it.